src/Analyzer.JsonRuleEngine/Schema/LeafExpressionDefinition.cs

```diff
@@ -104,5 +106,26 @@ public override Expression ToExpression()
 
             throw new NotImplementedException();
         }
+
+        /// <summary>
+        /// Validates the LeafExpressionDefinition for valid syntax
+        /// </summary>
+        internal override void Validate()
```

Comment:
I didn't see this called or tested anywhere.  Do we need this?  I know it's kinda weird because the structured expressions need one and this isn't using one, but I'm not sure of a good way to use it in the Converter.

Reply:
We don't need to call this. I can just leave this Validate empty. I added it to the ExpressionDefinition interface.